src/Hydrodynamics/SphSimulation.cpp

```diff
@@ -229,6 +229,25 @@ void SphSimulation<ndim>::ProcessParameters(void)
   }
 #endif
 
+
```

Comment:
I'd prefer a solution more like what I did with the dust: I think that we should set the supernova driver based on a single function call, rather than having to copy/paste this code everytime we need to add it somewhere.